### PR TITLE
Add notifications like FireProtection module

### PR DESCRIPTION
**Pull request by bjoern812**
Dear Maros,
I like the way that zway uses a notification style (email,pushover) in the module "FireProtection".
Using your great modules, especially the SecurityZone, I search for a way to use this "notification style",
so I take your EventWatcher and coded (more a copy&paste) the notification used in the "FireProtection"
module.
Maybe, you can have a look at it and if you like it, adopt it to your repo.

Thanks,
Björn

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ function EventWatcher (id, controller) {
     // Call superconstructor first (AutomationModule)
     EventWatcher.super_.call(this, id, controller);
     
-    this.eventCallback  = undefined;
+    this.CallbackEvent  = undefined;
```
Comment (maros) [inline index.js]: Shouldn't this be called 'callbackEvent' (lowecase 'c')

File: index.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /*** EventWatcher Z-Way HA module *******************************************
 
-Version: 1.04
+Version: 1.05
```
Comment (maros) [inline index.js]: Please don't increment the version. I have scripts for building releases that will get confused by manual changes

File: module.json [hunk 3]
```diff
@@ -201,5 +238,5 @@
       "type" : "object"
    },
    "singleton" : false,
-   "version" : "1.04"
+   "version" : "1.05"
```
Comment (maros) [inline module.json:242]: No manual version bumps please

File: module.json [hunk 4]
```diff
@@ -83,6 +83,27 @@
...
+                                 "device":{
+                                    "type": "select",
+                                    "label": "__notifications_helper__",
+                                    "datasource": "namespaces",
+                                    "field": "optionLabels",
+                                    "optionLabels": "namespaces:devices_toggleButton:notification:deviceName"
+                                 },
+                                 "message":{
+                                    "label":"__notifications_message__",
+                                    "type": "textarea"
```
Comment (maros) [inline module.json:101]: Maybe add a "helper" field that indicates that the message will only be used if the event does not have its own message

**Review by maros — CHANGES_REQUESTED** [review]
Thank you for your PR. I'd like to ask  you to make some minor amendments before I can finally accept and merge this PR

Thank you!

File: index.js (hunk 1 above)
Comment (bjoern812) [reply index.js]: Yes, of cource. You are right!

File: module.json (hunk 4 above)
Comment (bjoern812) [reply module.json:101]: Add a hellper field

File: module.json (hunk 3 above)
Comment (bjoern812) [reply module.json:242]: Reverse the change of the versions

**Comment by bjoern812** [comment]
Hi Maros,

thank you for your reply. I totally new to Github, so I changed my master branch and now I don't know
if I have todo? I see my changed files in this pull request. Must I create a new pull request or anything else?
Hope you may help me?

**Comment by maros** [comment]
Thank you for the PR. The PR was fine. For the future however, I'd recommend to make fewer commits (ie. not one commit per changed file)

**Pull request merged**